eve/workers/end2end/scripts/configs/zenkoversion.yaml

```diff
@@ -1,74 +1,73 @@
-# TODO: update image version to tagged versions
 ---
+# DO NOT EDIT - autogenerated file
 apiVersion: zenko.io/v1alpha1
 kind: ZenkoVersion
 metadata:
-  name: ${ZENKO_VERSION_NAME}
+  name: '${ZENKO_VERSION_NAME}'
 spec:
-  imagePullPolicy: Always
   versions:
     management:
       ui:
-        image: registry.scality.com/zenko-ui/zenko-ui
-        tag: '0.1.0'
+        image: '${ZENKO_UI_IMAGE}'
+        tag: '${ZENKO_UI_TAG}'
       api:
-        image: registry.scality.com/sf-eng/pensieve-api
-        tag:  '46e279b'
+        image: '${PENSIEVE_API_IMAGE}'
```

Comment:
How these variables get populated? (I might be missing something obvious.)

Reply:
the values are retrieve from `solution/deps.yaml` https://github.com/scality/Zenko/pull/1195/files#diff-a104984140e55eb80c2eea14bb0b3a260ed9ce9fcdc3b3c4335f0319265d5d89R14